nautobot/utilities/tests/test_filters.py

```diff
@@ -125,6 +126,7 @@ class DummyFilterSet(BaseFilterSet):
         multiplechoicefield = django_filters.MultipleChoiceFilter(field_name="choicefield")
         multivaluecharfield = MultiValueCharFilter(field_name="charfield")
         tagfield = TagFilter()
+        textfield = MultiValueCharFilter(field_name="textfield")
```

Comment:
This shouldn't be needed here, right? Isn't the whole idea of this PR that the correct filter class will get automatically chosen for the TextField?